Perpetuum.DataDumper/Factory/DataDumper.ModuleERP.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Perpetuum.DataDumper.Views;
+using Perpetuum.ExportedTypes;
+using Perpetuum.Items.Ammos;
+using Perpetuum.Modules;
+using Perpetuum.Modules.Weapons;
+using Perpetuum.Services.ExtensionService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Perpetuum.DataDumper.DataDumper;
+
+namespace Perpetuum.DataDumper {
+    public partial class DataDumper {
+        public ModuleERPDataView NewModuleERPDataView(Module module) {
+            var newView = new ModuleERPDataView();
+            InitModuleView(newView, module);
+
+            newView.module_recovery_chemical = module.GetBasePropertyModifier(AggregateField.chemical_damage_to_core_modifier).Value * 100;
+            newView.module_recovery_kinetic = module.GetBasePropertyModifier(AggregateField.kinetic_damage_to_core_modifier).Value * 100;
+            newView.module_recovery_seismic = module.GetBasePropertyModifier(AggregateField.explosive_damage_to_core_modifier).Value * 100;
+            newView.module_recovery_thermal = module.GetBasePropertyModifier(AggregateField.thermal_damage_to_core_modifier).Value * 100;
+            // module.GetBasePropertyModifier(AggregateField.heal)
```

Comment:
If this is "health" I think that is the repair state of the module/item.  Which for the data dump is likely irrelevant

Reply:
You're talking about the `.heal` attribute? It's just left over junk when I was looking for the properties. Basically each time I add a new item to dump I add the view, the standard factory, and then set a breakpoint to look around at how to grab what I need.

Reply:
oh gotcha, debugging code.  Should clean some of that up to not mislead others to think its something important.